src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -719,6 +731,12 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
     # Check Release Existance
     release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
 
+    utils.try_list_node_fix()
+    api_instance = kube_client.CoreV1Api()
+    node_api_response = utils.validate_node_api_response(api_instance, None)
+    is_arm64_cluster = check_arm64_node(node_api_response)
+    
+
     # Check forced delete flag
     if(force_delete):
```

Comment:
did we test force delete as well on arm64? current timeout is 20s for CRD deletion command, would be good to confirm if that works for arm64 too

Reply:
Yeah I tested that, it is working with the current changes